Formula/woboq_codebrowser.rb

```diff
@@ -0,0 +1,23 @@
+class WoboqCodebrowser < Formula
+  desc "Generate HTML from C++ Code"
+  homepage "https://code.woboq.org/"
+  url "https://github.com/woboq/woboq_codebrowser/archive/2.0.1.tar.gz"
+  sha256 "f2ec67bb49415986a4bbef515024009433756c83e14fa84f5acc40c83eec19e6"
+
+  depends_on "llvm"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", "-DCMAKE_BUILD_TYPE=Release", "-DLLVM_CONFIG_EXECUTABLE=#{Formula["llvm"].opt_bin}/llvm-config", "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON"
+    system "make"
+    mkdir prefix.to_s
+    mkdir bin.to_s
+    cp "indexgenerator/codebrowser_indexgenerator", bin.to_s
+    cp "generator/codebrowser_generator", bin.to_s
```

Comment:
`bin.install "indexgenerator/codebrowser_indexgenerator", "generator/codebrowser_generator"` replaces the above four lines.